rif/relay/launch.md

```diff
@@ -1,5 +0,0 @@
----
-layout: redirect
-permalink: /rif/relay/launch/
-redirect: /guides/rif-relay/launch/
----
```

Comment:
keep the redirect, just change where it points to

Reply:
updated